realParams:
  IMAGE_WIDTH: 320
  IMAGE_HEIGHT: 240
  LANE_IMAGE_WIDTH: 224
  LANE_IMAGE_HEIGHT: 224
  ENA: 33
  IN1: 21
  IN2: 22
  ENB: 32
  IN3: 26
  IN4: 24
  FREQUENCY: 50
  RATE: 30
  FORWARD_RANGE: 0.57
  STEERING_OFFSET: 38
  MIN_SPEED: 43
  TURNING_SPEED: 28
  STEERING_GAIN: 10
  THROTTLE: 27
  MIN_PWM_PERCENTAGE: 10
  MAX_PWM_PERCENTAGE: 100
  PERSON_PROB: 0.6
  TRAFFIC_LIGHT_PROB: 0.7
  STOP_SIGN_PROB: 0.9
  STOP_TIME: 5
  PERSON_WAIT_TIME: 2
  TIME_BETWEEN_STOP: 5
  MIN_STOP_AREA: 16000
  MIN_PERSON_AREA: 8000
  firstRedMask: [0, 170, 170, 10, 255, 255]
  secondRedMask: [170, 170, 170, 180, 255, 255]
  greenMask: [55, 199, 209, 133, 255, 255]
  modelPath: "/home/jetson/github/tfg-amariscal/models/road_following_model_real_trt.pth"
